### ISSUE TITLE
Hide normal push button

### ISSUE THREAD

**Issue by thimmy687** [gitextensions/GitExtensions.GerritPlugin#27]
As a user I want be able to hide the "normal" push button, if I use gerrit and this plugin, so that I don't accidently push it.

This should be effect the main gui and the commit form as well

**Comment by maraf** [comment]
There is no standard/nice way to do it. I'm modifying Git Extensions UI through enumeration of win forms control tree.

**Comment by pmiossec** [comment]
Here is a case that would benefit from the feature to be able to configure the toolbar by letting the user hide/show the buttons: 
https://github.com/gitextensions/gitextensions/issues/4952#issuecomment-715327377

**Comment by pmiossec** [comment]
@011V32 you're right GitExtensions now allows the user to hide the push button but it will be for all the repositories whereas in the PR #27 , it try to hide only when the repo is configured to use Gerrit.
